test/utils/generate-parsing-tests.ts

```diff
@@ -118,7 +118,8 @@ function createParsingTest<T extends TreeAdapterTypeMap>(
     };
 }
 
-const treePath = new URL('../data/html5lib-tests/tree-construction', import.meta.url);
+// TODO: Stop using the fork here.
```

Comment:
do we have an issue somewhere for this? would be good to mention the number i think

Reply:
Opened as https://github.com/inikulin/parse5/issues/448